apisix/plugin.lua

```diff
@@ -34,6 +34,9 @@ local type          = type
 local local_plugins = core.table.new(32, 0)
 local tostring      = tostring
 local error         = error
+-- make linter happy to avoid error: getting the Lua global "load"
```

Comment:
I guess this is no longer needed?

Reply:
This is needed. The lj-releng file just added to the keywords that can be used with the ignore comment.